internal/controllers/tcpedge_controller.go

```diff
@@ -261,6 +256,19 @@ func (r *TCPEdgeReconciler) findEdgeByBackendLabels(ctx context.Context, backend
 	return nil, iter.Err()
 }
 
+// Update the edge status and modules, called from both create and update.
+func (r *TCPEdgeReconciler) updateEdge(ctx context.Context, edge *ingressv1alpha1.TCPEdge, remoteEdge *ngrok.TCPEdge) error {
+	if err := r.updateEdgeStatus(ctx, edge, remoteEdge); err != nil {
+		return err
+	}
+
+	if err := r.updateIPRestrictionModule(ctx, edge, remoteEdge); err != nil {
```

Comment:
We could just `return r.updateIPRestrictionMoudle(ctx, edge, remoteEdge)` here right?

Reply:
Could, but I kinda like having the symmetry among the things-to-update :shrug: